src/components/Observations/index.tsx

```diff
@@ -1,3 +1,30 @@
-export default function Observations(): JSX.Element {
-  return <div>placeholder</div>;
+import { useCallback, useState } from 'react';
+import strings from 'src/strings';
+import { PlantingSite } from 'src/types/Tracking';
+import { APP_PATHS } from 'src/constants';
+import PlantsPrimaryPage from 'src/components/PlantsPrimaryPage';
+
+export default function PlantsDashboard(): JSX.Element {
+  const [selectedPlantingSite, setSelectedPlantingSite] = useState<PlantingSite>();
+  const [plantsSitePreferences, setPlantsSitePreferences] = useState<Record<string, unknown>>();
+
+  const onSelect = useCallback((site: PlantingSite) => setSelectedPlantingSite(site), [setSelectedPlantingSite]);
+  const onPreferences = useCallback(
+    (preferences: Record<string, unknown>) => setPlantsSitePreferences(preferences),
+    [setPlantsSitePreferences]
+  );
+
+  return (
+    <PlantsPrimaryPage
+      title={strings.OBSERVATIONS}
+      onSelect={onSelect}
+      pagePath={APP_PATHS.PLANTING_SITE_OBSERVATIONS}
+      lastVisitedPreferenceName='lastObservationsPlantingSite'
```

Comment:
Thoughts on namespacing these in a structured way to keep things organized? Such as `'<page>.<preferenceName>'`, e.g. `'observations.lastPlantingSite'`. Realize that this might break with how preferences have been organized in previous pages/components, if they are organized at all...

Reply:
That's a good suggestion, I understand the structure. I prefer to keep preference names flatter for now, let each component decide how they want to name the preferences. Also yes, this will break existing preferences that are already in the database. The preference names are not parsed for semantics anywhere.

Reply:
Here's a suggestion, since we aren't using plants dashboard in production yet.
I can make the preferences for the planting site pages hierarchical for sure. We can apply this going forward to other preferences.
For existing preferences (like accessions table, etc.) we will keep them as they are today.

Reply:
Sounds lovely. Thank you!